docs/modules/getting-started/pages/metrics.adoc

```diff
@@ -0,0 +1,69 @@
+= AeroGear Services Metrics SDK
+
+This document explains how to use and integrate the AeroGear Metrics SDK on a Cordova application.
+
+=== Install Dependencies
+
+Install the necessary dependencies in the app by running the following commands in your terminal:
+
+[source.bash]
+npm install @aerogear/app @aerogear/core
+cordova plugin add @aerogear/cordova-plugin-aerogear-metrics
+
+=== Deploy Metrics in Openshift
+
+Deploy an instance of Metrics from the Openshift console.
+
+=== Get the App Configuration
+
+The configuration needed to initialize the AeroGear SDK is contained in a JSON file generated by the https://github.com/aerogear/mobile-cli[AeroGear Mobile CLI]. It has the necessary parameters for every deployed service as well as other global ones. It should look like:
+
+```
+{
+  "version": 1,
+  "clusterName": "192.168.64.74:8443",
+  "namespace": "my-project",
+  "clientId": "my-app-id",
+  "services": [
+    {
+      "id": "metrics",
+      "url": "http://...",
+      ...
+    },
+    ...
+  ]
+}
+```
+
+=== Usage
+
+After dependencies are installed, call `init` from `@aerogear/app` when the app just starts:
```

Comment:
I will add that `Init method will configure and initialize all other AeroGear SDK's`

Reply:
IMO out of the scope, It's the metrics sdk's getting-started guide. Actually I'm thinking, maybe the best way would write a "getting started with the SDK", and then a small "usage" guide for each individual one. Think that otherwise we have to explain the `init` method in every pacakge (metrics getting-started, auth getting-started, push.. etc).

We could have a main page called "How to get started with SDK" or "Integrate AeroGear in your app" with these steps. This way also, we wouldn't include the plugin installation as required step (as you suggests in a comment above). @paolobueno @wtrocki WDYT?